src/index.html

```diff
@@ -7,8 +7,89 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+
+      <div class="logoMoyo">
+        <a
+        class="logo__link"
+        href="#homepage"
+        >
+          <img
+            class="imagine"
+            src="images/logo.png"
+            alt="logoMoyo"
+            >
+        </a>
+      </div>
+
+      <nav class="header__nav">
+        <ul class="header_nav_list">
+          <li class="nav_list__item">
+            <a
+            class="is-active nav__link"
+            href="#apple"
+            >
+            Apple
+            </a>
+          </li>
+          <li class="nav_list__item">
+            <a
+             class="nav__link" href="#samsung"
+            >
```

Comment:
```suggestion
            <a class="nav__link" href="#samsung">
```
https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md